src/frameworks/uikit/ui_touch.rs

```diff
@@ -16,6 +17,29 @@ use crate::objc::{
 use crate::window::Event;
 use crate::Environment;
 
+#[derive(Debug, Copy, Clone, Eq, PartialEq)]
+#[repr(i32)] // this should be an NSInteger!
+pub enum UITouchPhase {
+    UITouchPhaseBegan = 0,
+    UITouchPhaseMoved = 1,
+    UITouchPhaseEnded = 3,
+}
```

Comment:
For these C-like enums I do this instead (you can see this pattern in many files of touchHLE):

```rust
pub type UITouchPhase = NSInteger;
pub const UITouchPhaseBegan: UITouchPhase = 0;
// etc
```

It's a deviation from typical Rust but it avoids needless conversion back and forth, and acknowledges that C enums aren't really type-safe anyway. I should probably make a macro for this pattern so it's less repetitive.

Reply:
Done

It's actually simplified the code, I don't need to define GuestArgs anymore :)